src/NosCore.Parser/Parsers/MapNpcParser.cs

```diff
@@ -50,8 +52,8 @@ public void InsertMapNpcs(List<string[]> packetList)
             short map = 0;
             var npcs = new List<MapNpcDto>();
             var npcMvPacketsList = packetList.Where(o => o.Length > 14 && o[0].Equals("mv") && o[1].Equals("2") && long.Parse(o[2]) < 20000).GroupBy(s => s[2]).Select(s => Convert.ToInt32(s.First()[2])).ToList();
-            var effPacketsDictionary = packetList.Where(o => o[0].Equals("eff") && o[1].Equals("2") && long.Parse(o[2]) <= 20000).GroupBy(s => Convert.ToInt16(s[2])).ToDictionary(x=>x.Key, x => Convert.ToInt16(x.First()[3]));
-
+            var effPacketsDictionary = packetList.Where(o => o[0].Equals("eff") && o[1].Equals("2") && long.Parse(o[2]) <= 20000).GroupBy(s => Convert.ToInt16(s[2])).ToDictionary(x => x.Key, x => Convert.ToInt16(x.First()[3]));
+            var npcTalks = _npcTalkDao.LoadAll().ToDictionary(s => s.DialogId, s => s);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'npcTalks'.](https://app.codacy.com/manual/NosCoreIO/NosCore/pullRequest?prid=5306517)